.github/workflows/tests.yml

```diff
@@ -38,11 +40,14 @@ jobs:
                   -DSINGULARITY_USE_HELMHOLTZ=ON \
                   -DSINGULARITY_TEST_HELMHOLTZ=ON \
                   -DSINGULARITY_FORCE_SUBMODULE_MODE=ON \
+                  -DSINGULARITY_USE_V_AND_V_EOS=OFF \
```

Comment:
Removes `SAP_Polynomial`, `NobleAbel` and `CarnahanStarling` equations of state from the default variant. Down the line, we may also wish to remove `StiffGas` and the relativistic modifier.